File: Lógica de Programação/Linguagem C#/retangulo.cs
/*

Problema "retangulo" 
Fazer um programa para ler as medidas da base e altura de um retângulo. Em seguida, mostrar o valor 
da área, perímetro e diagonal deste retângulo, com quatro casas decimais, conforme exemplos. 
Exemplo 1: 
Base do retangulo: 4.0 
Altura do retangulo: 5.0 
AREA = 20.0000 
PERIMETRO = 18.0000 
DIAGONAL = 6.4031 


*/

using System;
using System.Globalization;
class HelloWorld {
  static void Main() {
   
   
    Console.Write("Base do retangulo: ");
    double basee = double.Parse(Console.ReadLine());
    Console.Write("Altura do retangulo: ");
    double altura = double.Parse(Console.ReadLine());
    
    double area = basee * altura;
    double perimetro = 2 * (basee + altura);
    double diagonal = Math.Sqrt(basee * basee + altura * altura);
    
    CultureInfo CI = CultureInfo.InvariantCulture;
    
    Console.WriteLine("AREA = " + area.ToString("F4", CI));
    Console.WriteLine("perimetro = " + perimetro.ToString("F4", CI));
    Console.WriteLine("DIAGONAL = " + diagonal.ToString("F4", CI));
    
    
  }
}

File: Lógica de Programação/Linguagem Python/retangulo.py
'''

                          Problema "retangulo" 
Fazer um programa para ler as medidas da base e altura de um retângulo. Em seguida, mostrar o valor 
da área, perímetro e diagonal deste retângulo, com quatro casas decimais, conforme exemplos. 
Exemplo 1: 
Base do retangulo: 4.0 
Altura do retangulo: 5.0 
AREA = 20.0000 
PERIMETRO = 18.0000 
DIAGONAL = 6.4031 
Exemplo 2: 
Base do retangulo: 10.3 
Altura do retangulo: 13.1 
AREA = 134.9300 
PERIMETRO = 46.8000 
DIAGONAL = 16.6643 

'''

import math

base = float(input("Base do retangulo: "))
altura = float(input("Altura do retangulo: "))

area = float(base * altura) 
perimetro = float(2 * (base + altura))
diagonal = float( math.sqrt(base * base + altura * altura))

print(f"AREA: {area:.4f}")
print(f"PERIMETRO: {perimetro:.4f}")
print(f"DIAGONAL: {diagonal:.4f}")